cmd/config/get/get.go

```diff
@@ -90,8 +97,24 @@ func PrintConfig(arg string) error {
 		fmt.Fprintf(Writer, "%s\n", cfg.URL)
 	case "fedramp":
 		fmt.Fprintf(Writer, "%v\n", cfg.FedRAMP)
+	case "keyrings":
```

Comment:
@tylercreller This isn't really reading the users config right? It's listing available keyrings that the user can use. That's a small, but different change in functionality to all of the other `rosa config get` commands and  I kinda feel like you've covered that in the `Prints the value of a config variable. See 'rosa config --help' for supported config variables.` output of this command if I run `rosa help config get`.

My personal expectation is that a `rosa config get keyring` would return the keyring I'm currently using (if that makes sense), but I would expect to read the list of available keyrings from docs or `rosa config help` command.

WDYT and also would be good to get some other opinions here rather than just me :-)

Reply:
@tylercreller Thanks for the explanation and I understand the intent.

Did you consider `rosa config set keyring $KEYSTORE` versus the environment variable implementation? It would restrict `rosa` to managing a single external environment variable (`OCM_CONFIG`) and might offer another option here in terms of helping the user see the list of supported keyrings e.g:

* `rosa config set keyring`
  * Outputs: "Selected keyring is not valid. Valid options are: "foo", "bar" 
* `rosa config set keyring doesntExist` 
  * Outputs: "Selected keyring 'doesntExist' is not valid. Valid options are: "foo", "bar" 
* `rosa config set keyring keychain`
* `rosa config get keyring`

This would leave the config file managing _only_ the selected keyring. The other values in the config are completely ignored. 

I don't want to over-engineering this/get too hung up on it. So let's get other opinions from other members of the team and make a decision.

Reply:
Always appreciate your thoughtful input @robpblake. We did originally consider an implementation like this. We opted to not go this route for similar consistency reasons. The keyring selection cannot be stored in the config file itself since the keyring stores the config data as an alternative to the file. This would be equivalent of storing the config file path in the config file itself. `rosa config set keyring` would still need to store that value in another file/variable/etc.

To maintain consistency we opted for the keyring env variable to match parity with standards set from `OCM_CONFIG` since they are mutually exclusive and handle similar operations.

Reply:
Thanks @tylercreller 

My current vote then based upon your explanation is to drop the `rosa config get keyrings` and rely on  documentation. I'd prefer consistency of the command semantics e.g `rosa config get` reads your actual applied config versus potential options for you.

If we get feedback that folks want to see available options for config values, then I think would could look at something like `rosa config defaults` or `rosa config describe $config` or some other route.

cheers,

Rob

Reply:
Updated based on our discussions @robpblake